src/RelayServer.ts

```diff
@@ -483,6 +483,20 @@ export class RelayServer extends EventEmitter {
         return maxPossibleGas;
     }
 
+    isSponsoredTx(req: RelayRequest | DeployRequest): boolean {
+        const { disableSponsoredTx, sponsoredDestinations } = this.config;
+
+        if (!disableSponsoredTx) {
+            return true;
+        }
+
+        if (sponsoredDestinations.includes(req.request.to)) {
+            return true;
+        }
```

Comment:
This seems to say that if sponsorship is disabled in config then check if request contains `to` field and if so allow sponsored tx anyway? 

```suggestion
        if (!disableSponsoredTx && sponsoredDestinations.includes(req.request.to)) {
            return true;
        }
```

Reply:
Updated, but its with an || conditional since it could be either, not both.